sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryParsingErrors.scala

```diff
@@ -392,10 +390,9 @@ object QueryParsingErrors extends QueryErrorsBase {
 
   def unexpectedFormatForResetConfigurationError(ctx: ResetConfigurationContext): Throwable = {
     new ParseException(
-      s"""
-         |Expected format is 'RESET' or 'RESET key'. If you want to include special characters
-         |in key, please use quotes, e.g., RESET `ke y`.
-       """.stripMargin.replaceAll("\n", " "), ctx)
```

Comment:
Besides the typo in "key", this way of formatting causes an extraneous space at the beginning and 7 extraneous spaces at the end.

Reply:
Note that other messages in this file besides the ones I changed do not end with any spaces.
For this reason, I add a space in `ParseException#getMessage` before the `(line $l, pos $p)`.